src/System.IO.Ports/src/Configurations.props

```diff
@@ -5,12 +5,12 @@
       netstandard-Linux;
       netstandard-OSX;
       netstandard;
-      net461;
+      net461-Windows_NT;
```

Comment:
Double check that the package still gets a RID-less asset.

Reply:
jep works as expected.

![image](https://user-images.githubusercontent.com/7412651/55508459-3ce1ff00-565a-11e9-9046-9d873dd56c75.png)